test/acceptance/README.md

```diff
@@ -13,3 +13,9 @@ These files are extensively commented, so check them out if you want to find out
 ## Getting HTTP logs
 By passing the `-debug-http` flag to the test suite (via `go test ./test/acceptance/ -args -debug-http`),
 you'll be able to see the HTTP requests and responses made by the test suite to Weaviate.
+
+## Updating the acceptance test expected schema
```

Comment:
I think we should get rid of the prototype over time, but great that you're documenting the process.